CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 
 namespace CompanyEmployees.Core.Domain.Entities
```

Comment:
The `Salary` property should be nullable (`decimal?`) if it's not required, or it should be explicitly validated elsewhere. Consider whether all employees must have a salary defined.

```suggestion
public decimal? Salary { get; set; }
```